src/main/resources/assets/admin/common/js/ui/uploader/UploaderEl.ts

```diff
@@ -766,8 +767,17 @@ export class UploaderEl<MODEL extends Equitable>
         }
 
         this.beforeSubmit();
+        const uploadItem = this.processFile(id, name);
+
+        if (this.config.allowMimeTypes != null && this.config.allowMimeTypes.length > 0) {
+            if (this.config.allowMimeTypes.filter(allowedMimeType => allowedMimeType === uploadItem.getFileType()).length === 0) {
```

Comment:
This can be 

`if (!this.config.allowMimeTypes.find(allowedMimeType => allowedMimeType === uploadItem.getFileType()))`